README.rst

```diff
@@ -389,6 +393,41 @@ Cats secrets out of a file::
 
     git secrets --add-provider -- cat /path/to/secret/file/patterns
 
+Options for ``--remove``
+~~~~~~~~~~~~~~~~~~~~~
+
+``--global``
+    Removes patterns from the global git config
+
+``-u, --deleteliteral``
+    Removes a literal pattern added with the --add command from the git config
+
+``-d, --deleteallowed``
+    Removes an allowed pattern from the git config
```

Comment:
I suggest using the same options as --add, namely -l/--literal and -a/--allowed. This will make it convenient to perform equivalent add and delete commands.